cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1211,9 +1211,9 @@ request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = | inputlookup cs_device_master_table \
 \
-| append [search `cs_lansweeper` | dedup id | rename id as lansweeper_id, site_name as Site, "asset.assetName" as Name, "asset.assetTypeName" as AssetType, "assetCustom.stateName" as AssetState, "asset.userDomain" as Domain, "asset.assetGroups{}.groupName" as GroupName, "asset.ipAddress" as IPAddress, "asset.mac" as lansweeper_mac_address, "asset.osCodeId" as OSVersion, "asset.userName" as lansweeper_user, "asset.buildNumber" as BuildNumber, "asset.version" as AssetVersion \
-| rename host as lansweeper_collected_by, _time as lansweeper_last_event, IPAddress as ip | eval host=lower(Name) \
-| table lansweeper_last_event, lansweeper_collected_by, host, ip, lansweeper_id, Site, AssetType, AssetState, Domain, GroupName, lansweeper_mac_address, OSVersion, BuildNumber, AssetVersion, lansweeper_user] \
+| append [search `cs_lansweeper` | eval id=coalesce(AssetID,id)| dedup id  | rename id as lansweeper_id, site_name as Site, AssetName as Name, AssetTypename as AssetType, Statename as AssetState, Userdomain as Domain, AssetGroup as GroupName, IPAddress as IPAddress, Mac as lansweeper_mac_address, OScode as OSVersion, Username as lansweeper_user, BuildNumber as BuildNumber, version as AssetVersion \
+| rename host as lansweeper_collected_by, _time as lansweeper_last_event, IPAddress as ip , OS as lansweeper_OS| eval host=lower(Name) \
+| table lansweeper_last_event, lansweeper_collected_by, host, ip, lansweeper_id, Site, AssetType, AssetState, Domain, GroupName, lansweeper_mac_address, OSVersion, BuildNumber, AssetVersion, lansweeper_user,lansweeper_OS, Description, IPLocation,FQDN ] \
```

Comment:
Need to make necessary changes to Device Master Table dashboard. 
Specifically the below line in the dashboard:
- | eval lansweeper_os =AssetType." ".OSVersion." ".AssetVersion | eval os=coalesce(tenable_os, QUALYS_OS, lansweeper_os)

Reply:
There is a savedsearch called "Device Master Table CleanUp". Check to see what changes we need there.